packages/paste-website/gatsby-node.js

```diff
@@ -0,0 +1,13 @@
+const {createFilePath} = require(`gatsby-source-filesystem`);
+
+exports.onCreateNode = ({node, getNode, actions}) => {
```

Comment:
Can you add some comments here explaining what this is or linking to some docs please

Reply:
@TheSisb I've added a comment explaining the what and why.